2014-12-24  Yaakov Selkowitz  <yselkowi@redhat.com>

	PR gas/17753
	* config/tc-mep.c (md_begin): Specify types of vararg literals.

--- a/gas/config/tc-mep.c	2014-10-14 02:32:03.000000000 -0500
+++ b/gas/config/tc-mep.c	2014-12-24 18:32:36.030124500 -0600
@@ -486,12 +486,12 @@ md_begin ()
   mep_cop = mep_config_map[mep_config_index].cpu_flag & EF_MEP_COP_MASK;
 
   /* Set the machine number and endian.  */
-  gas_cgen_cpu_desc = mep_cgen_cpu_open (CGEN_CPU_OPEN_MACHS, 0,
+  gas_cgen_cpu_desc = mep_cgen_cpu_open (CGEN_CPU_OPEN_MACHS, 0U,
 					 CGEN_CPU_OPEN_ENDIAN,
 					 target_big_endian
 					 ? CGEN_ENDIAN_BIG
 					 : CGEN_ENDIAN_LITTLE,
-					 CGEN_CPU_OPEN_ISAS, 0,
+					 CGEN_CPU_OPEN_ISAS, (CGEN_BITSET *)0,
 					 CGEN_CPU_OPEN_END);
   mep_cgen_init_asm (gas_cgen_cpu_desc);
 
